netboot-services/cleaner/main.go

```diff
@@ -166,6 +182,34 @@ func bytesToGiB(bytes float64) float64 {
 	return bytes / math.Pow(1024, 3)
 }
 
+func getDanglingKernelFilesOlderThanDate(folderName string, dayCount int) []fs.DirEntry {
```

Comment:
This function is never invoked

Reply:
As discussed:
The function is invoked in main _but_
I've the issue that I currently remove to many kernel-json files. I'll fix this issue and close the conversation afterwards.